signup-service/src/test/java/io/mosip/signup/services/RegistrationServiceTest.java

```diff
@@ -643,7 +643,7 @@ public void doVerifyChallenge_toResetPassword_thenKBAChallengeNotFound() {
     }
 
     @Test
-    public void doVerifyChallenge_toResetPassword_thenFetchIdentityFailed() {
+    public void doVerifyChallengeInResetPassword_thenFetchIdentityFailed_throwException() {
```

Comment:
can we rename this to "doVerifyChallengeInResetPassword_whenFetchIdentityFail_throwException()"